doc/release-notes.md

```diff
@@ -35,65 +28,109 @@ reindex or re-sync the whole chain.
+or not.
+- Exposed transaction version numbers are now treated as unsigned 16-bit integers instead of signed 16-bit integers.
+This matches their treatment in consensus logic. Versions greater than 3 continue to be non-standard (matching previous
+behavior of smaller than 1 or greater than 3 being non-standard). Note that this includes the joinpsbt command, which
+combines partially-signed transactions by selecting the highest version number.
+- New RPC `getrawtransactionmulti` that can return batch up to 100 raw transactions at one request.
+- The `getpeerinfo` RPC now returns a `connection_type` field. This indicates the type of connection established with
+the peer. It will return one of six options. For more information, see the `getpeerinfo` help documentation.
+- The `getpeerinfo` RPC now has additional `last_block` and `last_transaction` fields that return the UNIX epoch time
+of the last block and the last valid transaction received from each peer. (#19731)
+- `getblockchaininfo` now returns a new `time` field, that provides the chain tip time.
+- `fundrawtransaction` and `walletcreatefundedpsbt` when used with the `lockUnspents` argument now lock manually
+selected coins, in addition to automatically selected coins. Note that locked coins are never used in automatic coin
+selection, but can still be manually selected.
+- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee if the transaction passes
+validation. The `testmempoolaccept` RPC now accepts multiple transactions (still experimental at the moment, API may be
+unstable).
+
+## Added RPCs
+
+- `submitchainlock` RPC allows the submission of a ChainLock signature. Note: This RPC is whitelisted for the Platform
+RPC user.
+- `getassetunlockstatuses` RPC allows fetching of Asset Unlock txs by their withdrawal index. The RPC accepts an array
+of indexes and an optional block height. The possible outcomes per each index are: `chainlocked`, `mined`, `mempooled`,
+`unknown`. Note: If a specific block height is passed on request, then only `chainlocked` and `unknown` outcomes are
+possible. This RPC is whited for the Platform RPC user.
+- `quorum dkginfo` RPC returns information about DKGs: `active_dkgs` and `next_dkg`.
+
+## Docker Build System
+
+Modified the Dockerfile to exclude the `dash-qt` graphical wallet interface from the set of binaries copied to
+`/usr/local/bin` during the Docker image build process. This change streamlines the Docker image, making it more
+suitable for server environments and headless applications, where the graphical user interface of `dash-qt` is not
+required. The update enhances the Docker image's efficiency by reducing its size and minimizing unnecessary components.
 
-The miner has been disabled for Windows binaries built via Guix. This resolves
-an issue that was preventing users from installing v20.0.3 due to false
-positive alerts from Microsoft Defender and other antivirus providers.
+## Command-line options
```

Comment:
2 new options:
`-capturemessages` Capture all P2P messages to disk, see contrib/message-capture/message-capture-docs.md for more info
`-startupnotify=<cmd>` Execute command on startup.

Reply:
https://github.com/dashpay/dash/pull/5910/files#diff-e04cb1e9f29f76897c6b84334238f621ff45ba13cf180ad1d0d442d50ebe51f7R89-R90